website/docs/error-reference.md

```diff
@@ -936,6 +934,19 @@ srb tc --packager-layers util,lib,app
 If the flag is passed with no argument, then the default valid layers are
 `library` and `application`.
 
+## 3726
+
+> This error is specific to Stripe's custom `--stripe-packages` mode. If you are
+> at Stripe, please see [go/modularity](http://go/modularity) and
+> [go/layers](http://go/layers) for more.
+
+If a package is at `strict_dependencies` level `layered` or stricter, there are
+2 restrictions on what packages it may import:
+
+- all packages it imports must also be at `strict_dependencies` level `layered`
+  (or stricter)
```

Comment:
```suggestion
- all packages it imports must also be at `strict_dependencies 'layered'
  (or stricter)
```

Reply:
for context: we never say "At `# typed:` level `true` we run inference." Instead, we say, "At `# typed: true` we run inference."